resources/themes/STARTERTHEME/Gruntfile.js

```diff
@@ -13,10 +13,13 @@ module.exports = function (grunt) {
         files: [{
           expand: true,
           cwd: 'sass',
-          src: ['*.scss', '**/*.scss'],
+          src: '**/*.scss',
```

Comment:
We need to document this "feature" in the themes README.md file. Besides that and Simons comment: 10/10.